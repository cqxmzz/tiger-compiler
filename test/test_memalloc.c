#include "stdio.h"
#include "stdlib.h"
#include "coffgrader.h"
//
// Xiangru Chen
//
int main() {
  int i, pid, status;
  for (i = 0; i < 10; ++i) {
    pid = exec("hello.coff", 0, 0);
    assertTrue(pid > 0);
    assertTrue(join(pid, &status) == 1);
    assertTrue(status == 0);
  }
  printf("done.\n");

  done();

  return 0;
}
