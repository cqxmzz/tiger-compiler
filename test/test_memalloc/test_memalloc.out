hello nachos!hello nachos!hello nachos!hello nachos!hello nachos!hello nachos!hello nachos!hello nachos!hello nachos!hello nachos!done.
